test/e2e/framework/framework.go

```diff
@@ -123,9 +123,18 @@ func (f *Framework) BeforeEach() {
 
 	ginkgo.By("Creating kubernetes clients")
 
+	// We accept either KubeContexts + a single KubeConfig variable, or lots of individual
+	// KubeConfigs
+	Expect(len(TestContext.KubeConfigs) > 0 && len(TestContext.KubeContexts) > 0).NotTo(BeTrue())
```

Comment:
Can you add a user-friendly message otherwise the  error output will be vague, eg "Invalid kube configuration. You must specify either a single \"kubeconfig\" parameter containing  multiple kube config paths or multiple \"kubeconfig-path\" parameters, one for kube config path"?